Makefile

```diff
@@ -60,20 +64,20 @@ cpu=10
 gotest=go test -v ./tests/azure -parallel $(parallel) -cpu $(cpu)
 gotestargs=-args -logger $(logger)
 
-test: image
+test: imagedev
 	./hack/run.sh nash ./azure/vm_test.sh
 
-test-integration: image
+test-integration: imagedev
 	./hack/run.sh $(gotest) -timeout $(integration_timeout) -run=$(run) ./... $(gotestargs)
 
-test-examples: image
+test-examples: imagedev
 	./hack/run.sh $(gotest) -timeout $(examples_timeout) -tags=examples -run=TestExamples $(gotestargs)
 
 # It is recommended to use this locally. It takes too much time for the CI
-test-all: test
+test-all: testdev
```

Comment:
There is no `testdev` target. Did you mean `imagedev`?

Reply:
This target seems ancient =P...never use it

Reply:
Remembered and fixed it =P